### PR TITLE
DSD-3909: Make file for Python generation

### PR DESCRIPTION
**Pull request by creynolds15**

### CODE REVIEW

**Comment by ErinCompaan** [comment]
Looks reasonable to me.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+build/dependencies/*
```
Comment (tparikh) [inline .gitignore]: nit: please add a new line at the end of the file

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+	mkdir -p build/generated/python
+	protoc proto/tamr/api/v1beta1/*.proto \
+	    -I='proto' \
+	    -I='build/dependencies/googleapis' \
+	    -I='proto/protoc-gen-openapiv2/options' \
+	    --grpc_python_out='build/generated/python_grpc' \
+	    --python_out='build/generated/python'
+
+clean:
+	rm -rf build
```
Comment (tparikh) [inline Makefile]: nit: please add a new line at the end of the file

**Review by tparikh — DISMISSED** [review]

File: tamr_sdk/jobs/jobs_client.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        )
+        return self.stub.ListJobs(request)
+
+    def stop_job(self, job_id):
+        request = jobs.StopJobRequest(jobId=job_id)
+        return self.stub.StopJob(request)
+
+    def create_job(self, job_definition):
+        request = jobs.CreateJobRequest(job=job_definition)
+        return self.stub.CreateJob(request)
```
Comment (tparikh) [inline tamr_sdk/jobs/jobs_client.py]: nit: please add a new line at the end of the file

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,4 @@
+build/*
+.sdk_env/*
+__pycache__/
+.idea/*
```
Comment (tparikh) [inline .gitignore]: nit: please add a new line at the end of the file

File: requirements.txt [hunk 5]
```diff
@@ -0,0 +1,2 @@
+grpcio==1.62.2
+google-api-python-client==2.126.0
```
Comment (tparikh) [inline requirements.txt]: nit: please add a new line at the end of the file

**Comment by rubab2** [comment]
@creynolds15 Let's add a note/disclaimer in the readme that this is an in-development work-in-progress and not ready for use from outside of Tamr's network.

**Review by ErinCompaan — APPROVED** [review]

**Pull request merged**